src/main/java/frc/trigon/robot/misc/objectdetectioncamera/SimulationObjectDetectionCameraIO.java

```diff
@@ -2,36 +2,25 @@
 
 import edu.wpi.first.math.geometry.*;
 import frc.trigon.robot.RobotContainer;
-import frc.trigon.robot.commands.Commands;
+import frc.trigon.robot.commands.commandfactories.GeneralCommands;
 import org.littletonrobotics.junction.Logger;
 
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
 
 public class SimulationObjectDetectionCameraIO extends ObjectDetectionCameraIO {
     public static boolean HAS_OBJECTS = true;
-    private static final Rotation2d HORIZONTAL_FOV = Rotation2d.fromDegrees(75);
+    private static final Rotation2d CAMERA_HORIZONTAL_FOV = Rotation2d.fromDegrees(75);
     private static final double
-            MAXIMUM_DISTANCE_METERS = 5,
-            MINIMUM_DISTANCE_METERS = 0.05;
+            MAXIMUM_VISIBLE_DISTANCE_METERS = 5,
+            MINIMUM_VISIBLE_DISTANCE_METERS = 0.05;
     private static final double PICKING_UP_TOLERANCE_METERS = 0.3;
 
-    private final ArrayList<Translation2d> objectsOnField = new ArrayList<>(Arrays.asList(
-            new Translation2d(2.9, 7),
-            new Translation2d(2.9, 5.5),
-            new Translation2d(2.9, 4.1),
-            new Translation2d(8.3, 7.45),
-            new Translation2d(8.3, 5.75),
-            new Translation2d(8.3, 4.1),
-            new Translation2d(8.3, 2.45),
-            new Translation2d(8.3, 0.75),
-            new Translation2d(13.65, 7),
-            new Translation2d(13.65, 5.5),
-            new Translation2d(13.65, 4.1)
+    private final ArrayList<Translation2d> objectsOnField = new ArrayList<>(List.of(
+            new Translation2d(0, 0)// TODO: Set game piece positions
     ));
     private final String hostname;
-    private Pose3d[] heldObject = {new Pose3d()};
+    private Pose3d heldObject = null;
```

Comment:
There's a reason this was an array. It allows me to send an empty array when no object is visible.

Reply:
But why can't it just be a null value when no object is visible?

Reply:
Advantage kit doesn't work good with logging nulls